content/docs/command-reference/pull.md

```diff
@@ -167,17 +164,13 @@ $ dvc pull train.dvc
 > Please delete the `.dvc/cache` directory first (with `rm -Rf .dvc/cache`) to
 > follow this example if you tried the previous ones.
 
-Our [pipeline](/doc/command-reference/pipeline) has been setup with these
+Our [pipeline](/doc/command-reference/dag) has been setup with these
 [stages](/doc/command-reference/run):
 
-```dvc
-$ dvc pipeline show evaluate.dvc
-data/data.xml.dvc
-prepare.dvc
-featurize.dvc
-train.dvc
-evaluate.dvc
-```
+> prepare  
+> featurize  
+> train  
+> evaluate
```

Comment:
Why a block quote? This looks kind of hidden. Please follow my request which was a simple list in the paragraph above...

Reply:
Alright. Apologies for the previous commit. 
I've made the new updated submission.

Reply:
Thanks. Don't get me wrong I appreciate the initiative but we also need to reduce as much as possible the rounds of feedback so if there's a previous decision it's not a good idea to change it unless for some reason it results impossible to do.